base/client.jl

```diff
@@ -210,19 +210,36 @@ end
 
 # call include() on a file, ignoring if not found
 include_ifexists(mod::Module, path::AbstractString) = isfile(path) && include(mod, path)
-
+function ttyhascolor()
+    term_type = get(ENV, "TERM","")
+    startswith(term_type, "xterm") && return true
+    try
+        @static if Sys.KERNEL === :FreeBSD
+            return success("tput AF 0")
+        else
+            return success("tput setaf 0")
+        end
+    catch
+        return false
+    end
+end
 function exec_options(opts)
     if !isempty(ARGS)
         idxs = findall(x -> x == "--", ARGS)
         length(idxs) > 0 && deleteat!(ARGS, idxs[1])
     end
     quiet                 = (opts.quiet != 0)
     startup               = (opts.startupfile != 2)
-    history_file          = (opts.historyfile != 0)
+    history_file          = (opits.historyfile != 0)
```

Comment:
```suggestion
    history_file          = (opts.historyfile != 0)
```
Seems to just have been a mistake. This is what's causing the current CI failures.